tests/linux/2024.vncjew/__min__c.json

```diff
@@ -0,0 +1,1044 @@
+                    "ID": "net/url/embedded",
+                    "RuleName": "http_url"
+                },
+                {
+                    "Description": "Handles URL strings",
+                    "MatchStrings": [
+                        "RequestURI"
+                    ],
+                    "RiskScore": 1,
+                    "RiskLevel": "LOW",
+                    "RuleURL": "https://github.com/chainguard-dev/malcontent/blob/main/rules/net/url/parse.yara#url_handle",
+                    "ID": "net/url/parse",
+                    "RuleName": "url_handle"
+                },
+                {
+                    "Description": "requests resources via URL",
+                    "MatchStrings": [
+                        "net/url"
+                    ],
+                    "RiskScore": 2,
+                    "RiskLevel": "MEDIUM",
+                    "RuleURL": "https://github.com/chainguard-dev/malcontent/blob/main/rules/net/url/request.yara#requests_urls",
+                    "ID": "net/url/request",
+                    "RuleName": "requests_urls"
+                },
+                {
+                    "Description": "transfer data between file descriptors",
+                    "MatchStrings": [
+                        "sendfile",
+                        "syscall.Sendfile"
+                    ],
+                    "RiskScore": 1,
+                    "RiskLevel": "LOW",
+                    "RuleURL": "https://github.com/chainguard-dev/malcontent/blob/main/rules/os/fd/sendfile.yara#sendfile",
+                    "ReferenceURL": "https://man7.org/linux/man-pages/man2/sendfile.2.html",
+                    "ID": "os/fd/sendfile",
+                    "RuleName": "sendfile"
+                },
+                {
+                    "Description": "communicate with kernel services",
+                    "MatchStrings": [
+                        "netlink"
+                    ],
+                    "RiskScore": 1,
+                    "RiskLevel": "LOW",
+                    "RuleURL": "https://github.com/chainguard-dev/malcontent/blob/main/rules/os/kernel/netlink.yara#netlink",
+                    "ID": "os/kernel/netlink",
+                    "RuleName": "netlink"
+                },
+                {
+                    "Description": "references 'masscan', an asynchronous TCP port scanner",
+                    "MatchStrings": [
+                        "masscan"
+                    ],
+                    "RiskScore": 3,
+                    "RiskLevel": "HIGH",
+                    "RuleURL": "https://github.com/chainguard-dev/malcontent/blob/main/rules/sec-tool/net/masscan.yara#masscan",
+                    "ID": "sec-tool/net/masscan",
+                    "RuleName": "masscan"
+                }
+            ],
+            "RiskScore": 4,
+            "RiskLevel": "CRITICAL"
+        },
+        "/__min__c.~": {
```

Comment:
Since UPX scans will result in two reports, we'll need to use JSON to maintain deterministic ordering.